dev/plugins/hu.elte.txtuml.xtxtuml.ui/src/hu/elte/txtuml/xtxtuml/ui/launching/XtxtUMLExecutionLaunchAsTxtUMLAppShortcut.xtend

```diff
@@ -0,0 +1,11 @@
+package hu.elte.txtuml.xtxtuml.ui.launching;
+
+import org.eclipse.debug.core.DebugPlugin
+
+class XtxtUMLExecutionLaunchAsTxtUMLAppShortcut extends XtxtUMLExecutionLaunchShortcut {
+    
+    override getConfigurationType(){
+    	return  DebugPlugin.getDefault().getLaunchManager().getLaunchConfigurationTypes().get(23);
```

Comment:
Are you sure this is portable?

Reply:
I found a better solution, searching based on ID.